stdlib/float.mli

```diff
@@ -589,60 +597,239 @@ module Array : sig
 
 
   val map_to_array : (float -> 'a) -> t -> 'a array
-  (** [map_to_array f a] applies function [f] to all the elements of [a],
+  (** [map_to_array ~f a] applies function [f] to all the elements of [a],
       and builds an array with the results returned by [f]:
       [[| f a.(0); f a.(1); ...; f a.(length a - 1) |]]. *)
 
   val map_from_array : ('a -> float) -> 'a array -> t
-  (** [map_from_array f a] applies function [f] to all the elements of [a],
+  (** [map_from_array ~f a] applies function [f] to all the elements of [a],
       and builds a floatarray with the results returned by [f]. *)
 
+  (**/**)
+
   (** {2 Undocumented functions} *)
 
   (* These functions are for system use only. Do not call directly. *)
   external unsafe_get : t -> int -> float = "%floatarray_unsafe_get"
   external unsafe_set : t -> int -> float -> unit = "%floatarray_unsafe_set"
+
 end
 
 module ArrayLabels : sig
-
   type t = floatarray
+  (** The type of float arrays with packed representation.  @since 4.08.0 *)
```

Comment:
```suggestion
  (** The type of float arrays with packed representation.
      @since 4.08.0
    *)
```
(it's not your error, of course - but I recall seeing odoc grumbling about this!)

Reply:
Fixed.